applications/luci-app-advanced-reboot/luasrc/controller/advanced_reboot.lua

```diff
@@ -42,7 +42,7 @@ function get_partition_os_info(op_ubi)
 			cp_info = util.trim(util.exec('. /etc/os-release && echo "$OPENWRT_RELEASE"'))
 		end 
 	end
-	logger(i18n.translate("attempting to mount alternative partition") .. " (mtd" .. tostring(op_ubi) .. ")")
+	logger(i18n.translatef("attempting to mount alternative partition (mtd%s)", tostring(op_ubi))
```

Comment:
@stangri 
There seems to be one ")" missing at least on lines 45 and 54.
Did you test this?
Reference to #3903

Reply:
I thouught I did, must have edited file directly on the router and those two fixes didn't carry to the final file. Sorry about that and thanks for quick fix.